docs/conf.py

```diff
@@ -3,6 +3,9 @@
 import os
 import sys
 
```

Comment:
AFAIK, this is the default ellipsis marker. The declaration is, therefore, redundant and should be removed.